bin/pocs

```diff
@@ -34,6 +34,17 @@ if [ "${START}" = 'help' ] || [ "${START}" = '-h' ] || [ "${START}" = '--help' ]
 	exit 1
 fi
 
+#Patch could be improved by using for loops to test for all docker containers
+#in case the user manually kills pocs-shell. Also the if statement should be true
+#if the string doesn't have characters
+
+#exit script before trying to start containers if containers are running
+docker_ps=$(docker ps -q -f name=pocs-shell)
+if [ "$docker_ps" = "87d6fde2a139" ]; then 
```

Comment:
This is hard-coded to a specific container name but is the right approach.  Basically if it returns anything other than nothing than it is running something (how's that for a vague statement :grin:). 

In `bin/pocs-cmd` we just check for an empty string via:

```bash
if [ ! "$(docker ps -q -f name=${DOCKER_NAME})" ]; then
```

That should be good enough here. If we wanted to force certain versions we could use a specific id otherwise best to be generic.